clang/test/AST/ast-dump-type-callingconv.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -triple aarch64 -ast-dump -ast-dump-filter foo %s \
+// RUN: | FileCheck --strict-whitespace %s
+
+// CHECK: {{foo1 'void \(\) __attribute__\(\(device_kernel\)\)'$}}
```

Comment:
```suggestion
// CHECK: foo1 'void () __attribute__((device_kernel))'{{$}}
```
You don't need to apply the regex to the whole match, this makes it a little bit easier to read.